application/modules/front/views/collection/about/bukhari.php

```diff
@@ -1,7 +1,7 @@
 <div class=about style="font-size: 14px; width: 70%;">
 Ṣaḥīḥ al-Bukhārī is a collection of hadīth compiled by Abu Abdullāh Muhammad Ibn Ismā`īl al-Bukhārī(rahimahullāh). 
 His collection is recognized by the overwhelming majority of the Muslim world to be one of the most authentic 
-collections of the Sunnah of the Prophet (<img src="/images/sallallahu_alaihi_wa_sallam.png" height="18px">).
+collections of the Sunnah of the Prophet (<span id="pbuh_img"></span>).
```

Comment:
You can replace this image and the associated logic of turning it black vs white with just the Unicode for PBUH:
U+FDFA

Then your global text color change will handle this